draft-ietf-taps-transport-security.md

```diff
@@ -796,16 +796,16 @@ The following table lists the availability of the above-listed optional features
 |---
 | Protocol | MA | CM | SV | AFN | CX | SC |
 |:---------|:--:|:--:|:--:|:---:|:--:|:--:|
-| TLS      | S  | N\* | M | S   | S  | S
+| TLS      | S  | U\* | M | S   | S  | S
 | DTLS     | S  | S  | M  | S   | S  | S
 | IETF QUIC | S | S  | M  | S   | S  | S
 | IKEv2+ESP | M | S  | M  | ?   | S  | S
 | SRTP+DTLS | S | ?  | M  | S   | S  | S
-| SRTP+ZRTP | U | ?  | ?  | U   | U  | N
```

Comment:
For SRTP, session caching depends on DTLS. I didn't follow DTLS 1.3 development, but if the 0-RTT extensions from TLS 1.3 carried over, then they could presumably be used with SRTP.

Reply:
This line is for the SRTP that doesn't employ DTLS for handshake. It's unclear from the text whether it supports session caching, config extensions, or application-layer feature negotiation, so I guessed "no". @chris-wood do you know? Or given the structure with ZRTP as variant, should we even cover it in the table?

Reply:
I've reverted them to ? in my latest edit, which I'll push shortly.